.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+
+updates:
+  - package-ecosystem: 'npm' # See documentation for possible values
+    directory: '/' # Location of package manifests
+    schedule:
+      interval: 'daily'
+    labels:
+      - 'dependencies'
+      - 'automerge' # see Kodiak `merge.automerge_label`
```

Comment:
Does it mean Kodiak will automatically merge the PR when all required checks are OK?

Reply:
Yes, which includes at least one approval from us, see: https://kodiakhq.com/docs/recipes#configuring-dependabot-with-the-automerge-label